app/representers/api/msg/image_representer.rb

```diff
@@ -4,6 +4,12 @@
 # Representer for Announce messaging
 #
 class Api::Msg::ImageRepresenter < Api::BaseRepresenter
+  include NestedImage
+
+  def self_url(represented)
```

Comment:
The hal base representer will always try to add a self link, so this makes it not fail to generate.
Before it was adding one w/o error using the direct routes to the images.